integrations/pagerduty/webhook_processors/services.py

```diff
@@ -0,0 +1,44 @@
+from clients.pagerduty import PagerDutyClient
+from kinds import Kinds
+from webhook_processors.abstract import (
+    PagerdutyAbstractWebhookProcessor,
+)
+from port_ocean.core.handlers.port_app_config.models import ResourceConfig
+from port_ocean.core.handlers.webhook.webhook_event import (
+    EventPayload,
+    WebhookEvent,
+    WebhookEventRawResults,
+)
+
+
+class ServiceWebhookProcessor(PagerdutyAbstractWebhookProcessor):
+    async def should_process_event(self, event: WebhookEvent) -> bool:
+        client = PagerDutyClient.from_ocean_configuration()
+        return (
+            event.payload.get("event", {}).get("event_type")
+            in client.service_upsert_events
+            or event.payload.get("event", {}).get("event_type")
+            in client.service_delete_events
```

Comment:
I feel like its a bit redundant to initialize the client on every event received, I rather hold the "events" in a const outside of the client.